components/controls/Tooltip.tsx

```diff
@@ -157,6 +155,7 @@ export class TooltipInner extends React.Component<TooltipProps, State> {
     // eslint-disable-next-line react/no-find-dom-node
     const toggleNode = findDOMNode(this);
 
+    let measurements;
```

Comment:
Can we type as `Measurement | undefined` ? So that the return type of the method is correct too ?